packages/devextreme/js/ui/form/ui.form.js

```diff
@@ -1182,10 +1183,12 @@ const Form = Widget.inherit({
     },
 
     _refresh: function() {
-        const editorSelector = `.${FOCUSED_STATE_CLASS} > :not(.dx-dropdowneditor-input-wrapper) input,`
-            + ` .${FOCUSED_STATE_CLASS} textarea`;
+        const editorSelector = '.' + FOCUSED_STATE_CLASS + ' input, .' + FOCUSED_STATE_CLASS + ' textarea';
+        const $editors = this.$element().find(editorSelector).filter((idx, el) =>
+            !$(el).parents(UNREFRESHABLE_COMPONENT_WRAPPERS.join(', ')).length
+        );
 
-        eventsEngine.trigger(this.$element().find(editorSelector), 'change');
+        eventsEngine.trigger($editors, 'change');
```

Comment:
Also well know - when this 'trigger' call was added? What does it fix?

Reply:
It was fix of this https://isc.devexpress.com/internal/ticket/details/T404958